external/storm-jdbc/src/test/java/org/apache/storm/jdbc/common/JdbcClientTest.java

```diff
@@ -92,3 +115,27 @@ public void cleanup() {
         client.executeSql("drop table " + tableName);
     }
 }
+
+class MockConnectionProvider implements ConnectionProvider {
+
+    private Map<String, Object> configMap;
+
+    public MockConnectionProvider(Map<String, Object> mockCPConfigMap) {
```

Comment:
The class name could be chosen to be more descriptive -- ThrowingConnectionProvider?